rgw/v2/tests/s3_swift/multisite_configs/test_bucket_chown_reshard.yaml

```diff
@@ -0,0 +1,9 @@
+config:
```

Comment:
Can we add polarion-id and script details in the config file?

Reply:
Done